### PR TITLE
Added raw HDR shader

### PR DESCRIPTION
**Pull request by richardrobinson0924**
In ToneMapPostProcess, added an HDR shader which does not apply any transformations

### CODE REVIEW

File: Audio/DirectXTKAudio_Desktop_2015_DXSDK.vcxproj [hunk 1]
```diff
@@ -48,25 +48,25 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v140</PlatformToolset>
```
Comment (walbourn) [inline Audio/DirectXTKAudio_Desktop_2015_DXSDK.vcxproj]: You updated this to 2019 instead of using the 2019 files. I don't have a 2019 DXSDK version, but it could be added easily enough.

File: DirectXTK_Desktop_2017.vcxproj [hunk 2]
```diff
@@ -493,31 +493,31 @@
     <ProjectGuid>{E0B52AE7-E160-4D32-BF3F-910B785E5A8E}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>DirectXTK</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```
Comment (walbourn) [inline DirectXTK_Desktop_2017.vcxproj]: You should use ``DirectXTK_Desktop_2019.vcxproj`` instead which already has these changes.

File: Inc/Audio.h [hunk 3]
```diff
@@ -33,13 +33,13 @@
 #pragma comment(lib,"xaudio2.lib")
 #else
 // Using XAudio 2.7 requires the DirectX SDK
-#include <C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\Include\comdecl.h>
-#include <C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\Include\xaudio2.h>
-#include <C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\Include\xaudio2fx.h>
-#include <C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\Include\xapofx.h>
+#include <C:/Program Files (x86)/Microsoft DirectX SDK (June 2010)/Include/comdecl.h>
```
Comment (walbourn) [inline Inc/Audio.h]: Why did you change the slash style?

**Comment by walbourn** [comment]
Can you explain your scenario here a bit more?

**Comment by richardrobinson0924** [comment]
@walbourn apologies, I'll make the changes on `DirectXTK_Desktop_2019` and I'll make sure to not change any non-related files.

**Comment by walbourn** [comment]
@richardrobinson0924 : That's fine, I'm just trying to understand what the usage scenario is here for a buffer that's already in the HDR10 colorspace and only needs the paper-white scaling...

**Comment by richardrobinson0924** [comment]
Well, the usage is for displaying images that are already in the HDR 2084 colorspace, that do not require any transformations / tonemapping, to display n an HDR 2084 monitor

**Comment by walbourn** [comment]
I see where you are going here... Adding a shader is a bit more complex that you have done here for all the cases I need to cover (DX11, DX11 with 2 MRT for Xbox, and the same in the DirectX Tool Kit for DX12). This is the 'identity case' for [this issue](https://github.com/microsoft/DirectXTK/issues/119).

Rather than have to add another 3*3 shaders to the matrix (every tonemapper by every HDR colorspace for the Xbox MR

**Comment by walbourn** [comment]
Note that when I closed this, I meant to link it back to the work item https://github.com/microsoft/DirectXTK/issues/119

**Comment by walbourn** [comment]
Sorry this took so long to get in the mainline. It fell off my radar and then COVID. In any case, as of [this commit](https://github.com/microsoft/DirectXTK/commit/cbd344d0e247d4538af6c5997478db600b7ccd86) you can provide your own color space transform matrix including the identity matrix.
